docs/src/app/components/pages/components/TimePicker/ExampleInline.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TimePicker from 'material-ui/TimePicker';
+
+/**
+ * Inline TimePicker is displayed below the input, rather than as a modal dialog.
+ */
+const TimePickerExampleInline = () => (
+  <div>
```

Comment:
I've  kept the `div` there to put an `inline-landscape-datepicker` which I am working on right now.